docs/greenops/argo-server-deployment-example.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
```

Comment:
Move this to one of the GreenOps folders, we don't want to leave too much in the Argo Workflows repo

Reply:
Moved this file to the greenops repo: https://github.com/GreenOpsInc/greenops-enterprise/pull/60